src/scripts/generator_base.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import argparse
+import random
+import math
+import sys
+
+# Local packages
+import settings
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate # Function
+from constants import _constants
+
+# Generator base class, other generator classes will intherit from this class
+class Generator:
+    test_case = "" # Actual test case with validate function call
+    file_output = [] # All the test cases generated (without function definition)
+
+    # Function for printing debug messages if the user wants to (-q switch : quiet)
+    def debug(self, message, options):
+        if not options.q:
+            print(message)
+
+    # Function for generating a random false value integer
+    def random_false_int(self, expected_value, MIN, MAX):
+        # Random number which must not be equal with the expected value
+        if ((random.randint(0, 1) and expected_value != MAX) or expected_value == MIN):
+            return random.randint(int(expected_value + 1), MAX)
+        return random.randint(MIN, int(expected_value - 1))
+
+    # Generating false numbers and returning with an array
+    def generate_false_numbers(self, expected_value, false_values_count, MIN, MAX):
+        false_values = []
+        for j in range(false_values_count):
+            false_values.append(self.random_false_int(expected_value, MIN, MAX))
+        return false_values
+
+    # Creating a string which contains the array of false values
```

Comment:
`Creating a string which contains the array of false values` -> `Stringify an array of false values`